build.sbt

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-ThisBuild / organization := "za.co.absa.fa-db"
+import Dependencies._
+
+ThisBuild / name := "fa-db"
+ThisBuild / organization := "za.co.absa"
```

Comment:
Will this not change the path from `za.co.absa.fa-db` to `za.co.absa`? 
So paths will ow be `za.co.absa.core_2.12` and `za.co.absa.slick_2.12`

Reply:
You are right. 👍  Actually not sure, how the change slipped in. 😊